device/iot-device-samples/send-batch-events/README.md

```diff
@@ -0,0 +1,60 @@
+# Send Event Sample
```

Comment:
Is this a copy-paste error? It should be "Send Batch Events Sample"

Reply:
See below, but it looks like you edited the wrong readme file

Reply:
yes it is